integr8ly/keycloak-client
# switch from internal to external url

The Keycloak CR previously only had an Status.InternalURL field.
It now has Status.InternalURL and Status.ExternalURL

Previously the Status.InternalURL pointed at the external route. 
With the Latest version, 13.0.0 , Status.InternalURL points at the service and 
Status.ExternalURL points at the route.

Change method AuthenticatedClient() to return the Status.ExternalURL i.e. the route
to keep it consistent with previous versions. 

Also, update go.mod to use keycloak-operator 13.0.0

**To Verify**
Run this version of the client in the integreatly operator
Ensure RHSSO user and cluster installs and works as expected
